### PR TITLE
add a libreoffice tc#1503827 for x11regression

### PR DESCRIPTION
**Pull request by chuchingkai**
Testing Result:

http://147.2.212.239/tests/262

### CODE REVIEW

File: tests/x11regressions/libreoffice/libreoffice_mainmenu_components.pm [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+# Case 1503827 - LibreOffice: Launch application components from system menu
+
+# open desktop mainmenu and click office
+sub open_mainmenu() {
+    my $self = shift;
+
+    wait_still_screen;
+    send_key "alt-f1";
+    assert_screen 'test-desktop_mainmenu-1', 20;
```
Comment (sysrich) [inline tests/x11regressions/libreoffice/libreoffice_mainmenu_components.pm]: Why the 20 second assert_screens on this and other lines? Is there something wrong with the default timeout (which is 30 seconds)?

Good Practice is to leave the timeout alone unless there is a very good reason why you must match it faster.. and I do not see this in this case

*NOTE* This isn't critical and I might still merge the test anyway, but I'd much rather it be perfect and follow good practices

File: tests/x11regressions/libreoffice/libreoffice_mainmenu_components.pm [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    $self->open_mainmenu();
+    assert_and_click 'mainmenu-office-lo';    #open lo
+    assert_screen 'welcome-to-libreoffice';
+    send_key "ctrl-q";                        #close lo
+
+    $self->open_mainmenu();
+    assert_and_click 'mainmenu-office-base';    #open base
+    assert_screen 'oobase-select-database', 45;
+    send_key "ret";
+    sleep 3;
```
Comment (sysrich) [inline tests/x11regressions/libreoffice/libreoffice_mainmenu_components.pm]: Sleep is not necessary, the assert_screen will take care of the delay you want here

File: tests/x11regressions/libreoffice/libreoffice_mainmenu_components.pm [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    send_key "ctrl-q";                             #close writer
+
+    # launch components from Activities overview
+    $self->open_overview();
+    type_string "base";                            #open base
+    assert_screen 'overview-office-base';
+    send_key "ret";
+    assert_screen 'oobase-select-database', 45;
+    send_key "ret";
+    sleep 3;
```
Comment (sysrich) [inline tests/x11regressions/libreoffice/libreoffice_mainmenu_components.pm]: another unnecessary sleep. There is no need to use sleep if the next thing you are doing in the test is an assert_screen

**Comment by chuchingkai** [comment]
Thanks for your suggestion, the code has been fixed.

**Pull request merged**